src/librustc_mir/pretty.rs

```diff
@@ -92,7 +92,7 @@ pub fn write_mir_pretty<'a, 'b, 'tcx, I>(tcx: TyCtxt<'b, 'tcx, 'tcx>,
     where I: Iterator<Item=DefId>, 'tcx: 'a
 {
     let mut first = true;
-    for def_id in iter {
+    for def_id in iter.into_iter().filter(DefId::is_local) {
```

Comment:
Oops... I've built up too much of a reflex going from `in collection` to `in collection.into_iter().foo()`.